environments/staging/terraform.yml

```diff
@@ -246,25 +246,25 @@ servers:
     group: "couchdb2"
     os: bionic
 
-  - server_name: "rabbit_a2-staging"
+  - server_name: "rabbit_a3-staging"
     server_instance_type: "t3.large"
     network_tier: "app-private"
     az: "a"
     volume_size: 80
     volume_encrypted: yes
     volume_type: "gp3"
     group: "rabbitmq"
-    os: bionic
+    os: jammy
 
-  - server_name: "rabbit_b1-staging"
+  - server_name: "rabbit_b2-staging"
     server_instance_type: "t3.large"
```

Comment:
I just noticed in reviewing these PRs that the staging rabbit instances are larger (t3.large, 80GB disk) than the india instances (t3.small, 30 GB disk) which is surprising! I'll make a note of it in our cost saving sheet, but keep an eye out for these kinds of opportunistic improvements!